bootstrap.sh

```diff
@@ -1,54 +1,67 @@
-#!/bin/bash
+#!/usr/bin/bash
```

Comment:
Use `#!/usr/bin/env bash` for more portability, and searches users `PATH` for bash